auth/store_test.go

```diff
@@ -34,32 +34,35 @@ func dummyIndexWaiter(index uint64) <-chan struct{} {
 	return ch
 }
 
-func TestUserAdd(t *testing.T) {
+func setupAuthStore(t *testing.T) (*authStore, func(t *testing.T)) {
```

Comment:
let use named return here? func(t *testing.T) is very unclear. (store *authStore, teardownfunc func(t *testing.T))